common/flatpak-run.c

```diff
@@ -923,6 +923,19 @@ flatpak_run_add_pulseaudio_args (FlatpakBwrap         *bwrap,
     flatpak_bwrap_add_args (bwrap, "--dev-bind", "/dev/snd", "/dev/snd", NULL);
 }
 
+static void
+flatpak_run_add_gssproxy_args (FlatpakBwrap *bwrap)
+{
+  /* We only expose the gssproxy user service. The gssproxy system service is
+   * not intended to be exposed to sandboxed environments.
+   */
+  g_autofree char *gssproxy_host_dir = g_build_filename (g_get_user_runtime_dir (), "gssproxy", NULL);
+  const char *gssproxy_sandboxed_dir = "/var/lib/gssproxy/";
+
+  if (g_file_test (gssproxy_host_dir, G_FILE_TEST_EXISTS))
+    flatpak_bwrap_add_args (bwrap, "--bind", gssproxy_host_dir, gssproxy_sandboxed_dir, NULL);
```

Comment:
Should this be `--ro-bind`? I think you can connect to sockets that are on a read-only filesystem, but we probably don't want sandboxed apps to be allowed to create and delete files, sockets, symlinks, etc. in the host's `$XDG_RUNTIME_DIR/gssproxy`.

Reply:
True, that seems more secure.

Reply:
Hm...  I'm not sure why I didn't use `ro-bind` here. If it works, then yes, of course that seems better. Hopefully I was incorrectly thinking "need to write to sockets, better not be read-only" and there's no good reason for this to be writable.

I will test and find out.

Reply:
I've changed this to `--ro-bind` but still need to test to make sure it works. Leaving this MR in draft state until I have tested it.

Reply:
This works.